### ISSUE TITLE
msg.stack is undefined

### ISSUE THREAD

**Issue by ggregoire** [ajbrown/angular-loggly-logger#12]
Hello!

Line 204:

```javascript
var msg = args.length == 1 ? args[0] : args;
```

If I log the `msg` variable:

![loggly](https://cloud.githubusercontent.com/assets/591804/6127516/1e8f2032-b12c-11e4-931c-8272c0dc772c.png)

Then the stack is not in `msg` but in `msg[0]`.

I added a quick fix:

```javascript
if (angular.isDefined(msg.stack || msg[0].stack)) {
    //handling console errors
    if (logger.sendConsoleErrors() === true) {
        sending.message = msg.message || msg[0].message;
        sending.stack = msg.stack || msg[0].stack;
    } else {
        return;
    }
}
```

**Comment by ajbrown** [comment]
This is great! :star2:  Would you mind opening a pull request with the fix, so you're permanently credited in the history?

**Comment by ajbrown** [comment]
fixed by #14, thanks! :+1: :+1:
